glass/user.py

```diff
@@ -17,12 +17,24 @@
 
 """  # noqa: D205, D400
 
+from __future__ import annotations
+
+import typing
 from contextlib import contextmanager
 
 import numpy as np
+import numpy.typing as npt
+
+if typing.TYPE_CHECKING:
+    import collections.abc
+    import importlib.util
+    import pathlib
+
+    if importlib.util.find_spec("fitsio") is not None:
+        import fitsio
```

Comment:
This is a bit hacky, but seems to work for `mypy`

Reply:
But what's the point of the conditional import here?

Reply:
If I directly import `fitsio` then users _have_ to have `fistio` which you're trying to avoid?